integration/plugin/mountpoint/mountpoint_unix_test.go

```diff
@@ -723,19 +719,19 @@ func TestMountPointPluginKill(c *testing.T) {
 	}
 	ctrl[1].attachRes = ctrl[0].attachRes
 
-	id, err := request.RunV(client, []string{"/:/host"}, "busybox", []string{"top"})
+	id, err := containerRunV(client, []string{"/:/host"}, "busybox", []string{"top"})
 	require.Nil(c, err)
 
-	err = request.Kill(client, id)
+	err = containerKill(client, id)
 	require.Nil(c, err)
 
 	require.Equal(c, []string{"0:properties", "1:properties", "0:attach", "1:attach", "1:detach", "0:detach"}, events)
 }
 
 func TestMountPointPluginOOM(c *testing.T) {
-	daemonIsLinux(c)
-	memoryLimitSupport(c)
-	swapMemorySupport(c)
+	skip.IfCondition(c, testEnv.DaemonInfo.OSType != "linux")
+	skip.IfCondition(c, !sysInfo.MemoryLimit)
+	skip.IfCondition(c, !sysInfo.SwapLimit)
```

Comment:
This is checking `sysInfo` for the host where the tests are running, but the docker engine may be running on a different host. Does this test need to check `testEnv.IsLocalDaemon()` as well?

Reply:
I cribbed this from <https://github.com/moby/moby/blob/master/integration-cli/docker_cli_oom_killed_test.go#L11>. I'm not aware of additional requirements for the tests to support non local daemons. I can add the check if you'd like.